src/plot/series/custom-svg-series.js

```diff
@@ -146,13 +150,24 @@ class CustomSVGSeries extends AbstractSeries {
         positionInPixels,
         defaultType: customComponent,
         positionFunctions: {x, y},
-        style
+        style,
+        propsSize: size
       });
       return (
         <g
           className="rv-xy-plot__series--custom-svg"
           key={`rv-xy-plot__series--custom-svg-${index}`}
           transform={`translate(${positionInPixels.x},${positionInPixels.y})`}
+          onMouseOver={() => {
```

Comment:
I think you can simplify this a little bit? Check out the [mark series code](https://github.com/uber/react-vis/blob/master/src/plot/series/mark-series.js) maybe?

Reply:
Fixed to match `MarkSeries`